src/main/handlers/helpers/transcriptionEngines/voskTranscribeFunction.ts

```diff
@@ -0,0 +1,55 @@
+import { JSONTranscription } from 'main/types';
+import path from 'path';
+import fs from 'fs';
+import getVoskTranscript from '../../../../vosk';
+import { getAudioExtractPath } from '../../../util';
+import { TranscriptionFunction } from '../transcribeTypes';
+
+interface VoskWord {
+  end: number;
+  start: number;
+  word: string;
+}
+
+const transcribeWithVosk = async (audioFilePath: string) => {
+  const modelName = 'vosk-model-en-us-0.22';
+  const modelPath = path.join(
+    __dirname,
+    '../../../../../assets/voskModel',
+    modelName
+  );
+
+  if (!fs.existsSync(modelPath)) {
+    throw new Error(
+      `Model ${modelPath} not found. Have you downloaded the model and put it in the assets folder?`
+    );
+  }
+  const result = (await getVoskTranscript(modelPath, audioFilePath)) as string;
+
+  return result;
+};
+
+const transcriptionAdaptor: (wordList: VoskWord[]) => JSONTranscription = (
+  wordList
+) => ({
+  words: wordList.map((result) => ({
+    word: result.word,
+    startTime: result.start,
+    duration: result.end - result.start,
+    confidence: 0.6,
```

Comment:
If vosk doesn't output confidence just set confidence to `1` so that there are no underlines regardless of the threshold

Reply:
Done! Thanks